sale_timesheet_task_exclude/readme/USAGE.rst

```diff
@@ -1,4 +1,4 @@
+To set a Task as non billable:
 
-# Go to *Project > All Tasks*
-# Open specific task for editing
-# Check *Exclude From Billing*
+  # Go to *Project > All Tasks*, and open a specific Task form
```

Comment:
@dreispt here as well, `#.` (that's a mistake in original file) and no spaces before otherwise extra indent